scripts/aggregate-themes/disclaimer.txt

```diff
@@ -0,0 +1,7 @@
+/*
+ * Warning - Do not modify this file.
+ *
+ * This file is autogenerated and should not be checked into source control.
+ *
+ * It is recommended to add this file to your .gitignore and .eslintignore.
```

Comment:
also to `.dockerignore` and `.npmignore`

Reply:
These have been added: https://github.com/cerner/terra-toolkit/pull/252/commits/6ae2c94b5d48b535ead8ee6753bc7ea47f107578